src/swoop/api/models/__init__.py

```diff
@@ -295,9 +295,33 @@ class Events(BaseModel):
     pass
 
 
+class PayloadDetails(BaseModel):
+    payload_uuid: str | None = None
```

Comment:
This doesn't seem particularly detailed; perhaps a better name might be `PayloadSummary`?

Reply:
This should probably have an href to its detailed record, i.e., `/payloads/<uuid>`.

Reply:
is the href supposed to be a Link object or just a string?

Reply:
I think we should be taking inspiration from the STAC spec in our attempts to make the API browsable. So here's an item I'm looking at as an example: https://earth-search.aws.element84.com/v1/collections/sentinel-2-l2a/items/S2B_33UYT_20230601_0_L2A.

Notice that the item has `links`, but also has `assets`, and both are link-ish in that they have an `href` and `type`. But the assets don't have a `rel` property because it is implied that they are assets, whereas the `links` have to have a `rel` to define the kind of link.

So, is this a link object? Probably not. But is simply having an `href` all we need? Also probably not: it seems that we should specify `type` as we see in the asset example.